airbyte-integrations/connectors/source-surveymonkey/unit_tests/unit_test.py

```diff
@@ -0,0 +1,56 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+import json
+
+import pendulum
+import vcr
+from airbyte_cdk.models import SyncMode
+from source_surveymonkey import SourceSurveymonkey
+
+
+def test_get_updated_state():
+    source = SourceSurveymonkey()
+    config = json.load(open("secrets/config.json"))
+    streams = source.streams(config=config)
+    surveys_instance = [i for i in streams if i.__class__.__name__ == "Surveys"][0]
+    survey_responses_instance = [i for i in streams if i.__class__.__name__ == "SurveyResponses"][0]
+    with vcr.use_cassette("pretty_name.yaml", record_mode="new_episodes"):
```

Comment:
to make it a unit test you need to either:
1. mock requests or other "network" calls inside `get_updated_state`
2. or, like you did here, use vcr, but you should not record any new episodes but rather assert on all network calls that is not in pretty_name.yaml file (btw, the name is not pretty, it is confusing). Of course you need to commit yaml file as well if you use vcr.

I vote for option 1, as it is easier to maintain